data_science/health_monitoring_dashboard/healthdashboard.ipynb

```diff
@@ -0,0 +1,334 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "id": "7ef11301-3200-413e-9369-a079255166a0",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "\n",
+       "        <iframe\n",
+       "            width=\"100%\"\n",
+       "            height=\"650\"\n",
+       "            src=\"http://127.0.0.1:8052/\"\n",
+       "            frameborder=\"0\"\n",
+       "            allowfullscreen\n",
+       "            \n",
+       "        ></iframe>\n",
+       "        "
+      ],
+      "text/plain": [
+       "<IPython.lib.display.IFrame at 0x12187c1a0>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import dash\n",
+    "from dash import dcc\n",
+    "from dash import html\n",
+    "import plotly.express as px\n",
+    "import plotly.graph_objs as go\n",
+    "import plotly.figure_factory as ff\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "\n",
+    "# Load the data\n",
+    "file_path = '/Users/aryan.vadera/Desktop/CapstoneProjectA/HealthData.csv'\n",
```

Comment:
Can you please remove this file path as it points to a file on your local computer. Instead can you add the file in a "data" folder on GitHub and add a path to that.